blog.html

```diff
@@ -5,6 +5,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>BINABOX-BLOGPAGE</title>
     <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="styles.css" />
```

Comment:
I suppose one of lines above is redundant. Also you don't have file named styles.css in your project